loopin.go

```diff
@@ -48,8 +48,12 @@ var (
 type loopInSwap struct {
 	swapKit
 
+	executeConfig
```

Comment:
For the commit title prefix, we try to mention subsystems. (refactor)

Reply:
done